linux/datadog_wrapper

```diff
@@ -133,10 +133,18 @@ setUpDotnetEnv() {
         return
     fi
 
+    echo "Exporting environment variables"
     export CORECLR_ENABLE_PROFILING=1
     export CORECLR_PROFILER="{846F5F1C-F9AE-4B07-969E-05C26BC060D8}"
     export CORECLR_PROFILER_PATH="${DD_DIR}/Datadog.Trace.ClrProfiler.Native.so"
     export DD_DOTNET_TRACER_HOME="${DD_DIR}"
+
+    if [[ "${DD_PROFILING_ENABLED:-,,}" == "1" ]]; then
+        echo "Profiler is enabled."
+        export LD_PRELOAD="${DD_DIR}/continuousprofiler/Datadog.Linux.ApiWrapper.x64.so"
```

Comment:
I'm wondering if we definitely want to overwrite LD_PRELOAD or "append" to it? (I don't know if that's a thing for LD_PRELOAD)